### PR TITLE
Search index position in java

### PR DESCRIPTION
**Pull request by rutujadhanawade**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added code for Search insert position in java.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1119

### CODE REVIEW

**Comment by rutujadhanawade** [comment]
@iamrajiv @atarax665 @plazzy99 Please review.

File: Java/other/Search_insert_index.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        Scanner sc = new Scanner(System.in);
+        int len, target,index;
+        System.out.println("Enter the length of array");
+        len = sc.nextInt();
+
+	int arr[] = new int[len];
+        System.out.println("Enter the array elements");
+        
+        for (int i = 0; i < len; i++)
+            arr[i] = sc.nextInt();
```
Comment (vatsalkesarwani12) [inline Java/other/Search_insert_index.java:34]: remove extra lines spaces

Comment (vatsalkesarwani12) [inline Java/other/Search_insert_index.java:34]: make proper indentation

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]

**Comment by rutujadhanawade** [comment]
Done :+1: @plazzy99 Please review.

**Review by iamrajiv — APPROVED** [review]

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻

**Pull request merged**